solidity/dashboard/src/services/liquidity-rewards.js

```diff
@@ -0,0 +1,34 @@
+import { Web3Loaded, createERC20Contract } from "../contracts"
+
+// lp contract address -> wrapped ERC20 token as web3 contract instance
+const LPRewardsToWrappedTokenCache = {}
+
+// TODO implement functions
```

Comment:
The functions are implemented so I think we can remove this TODO 😉

Reply:
b09f05e